UPGRADE-2.0.md

```diff
@@ -35,7 +35,11 @@ The following exceptions have been removed:
 Please use their equivalent (same name) from the following namespace:
 `Gnugat\Redaktilo\Exception`.
 
-## File removal (1.6)
+## PHP and Number Search Removal (1.7)
 
-* `File` has been removed, use `Text` instead
-* `Editor#save`'s `filename` argument is now mandatory
+The following classes have been removed:
+
+* `LineNumberSearchStrategy`
```

Comment:
Maybe mention how to achieve this, maybe like this you did on the `@deprecated` phpdoc of the class:
`Use Text#setCurrentLineNumber instead`